File: QE-selenium/python/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as dirver:
    dirver.get("https://training-support.net/webelements/login-form/")
    print(dirver.title)
    dirver.find_element(By.XPATH,"//*[@id='username']").send_keys("admin")
    dirver.find_element(By.XPATH,"//*[@id='password']").send_keys("password")
    dirver.find_element(By.XPATH,"/html/body/div/main/div/div/div/div/div[2]/form/button").click()
    dirver.quit()

File: QE-selenium/java/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
     public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    driver.get("https://training-support.net/webelements/login-form/");
    System.out.println(driver.getTitle());
    driver.findElement(By.xpath("//*[@id=\"username\"]")).sendKeys("admin");
    driver.findElement(By.xpath("//*[@id=\"password\"]")).sendKeys("password");
    driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/form/button")).click();
    driver.quit();
}
}
